### PR TITLE
Changes Stypic and Silver Sulfadine to obey sanity

### PR DESCRIPTION
**Pull request by ghost**
# Previously

Stypic powder and Silver Sulfadine were the two best, easiest to make, healing chemicals in the game. They instantly healed brute and burn on application via patch (or splashing it on yourself) for the amount present, aND continued to treat it for -2 per tick as long as it was in you.

So a single brute patch from a medkit with a mob for a normal reagent metabolism would treat 50 damage immediately, and treat 500 brute damage over time at a rate better than omnizine or most other anti-damage chemicals such as bicardine, tricordazine, or pentetic acid.
# Now

Stypic powder and silver sulfadine will instantly heal an equal amount of brute or burn damage on application to mob via touch as they have in them, as they do now, and nothing else. They no longer heal over time for a ridiculous 500 damage or more.
# TL;DR for the stupid

Stypic powder and silver sulfadine will still work as they do now on application, instantly treating damage, but not healing over time.
# FAQ for the people who don't want this

fuck off
#### Changelog

:cl:
rscadd: Stypic Powder and Silver Sulfadine now obey physics and no longer expend 5x more energy than was used to create them in treating wounds.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
do it the other way around, instantly applying affect is dum

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -265,7 +254,7 @@
 	metabolization_rate = 0.5 * REAGENTS_METABOLISM
 
 /datum/reagent/medicine/salglu_solution/on_mob_life(mob/living/M)
-	if(prob(33))
+	if(prob(66))
```
Comment (ShadowDeath6) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:257]: !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!undocumented chang es !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Comment (Super3222) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Bad, Adam, Bad.

Comment (Super3222) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:257]: again, adam?

**Comment by ghost** [comment]
eph did it the other way around shadow

it was bad

**Comment by X-TheDark** [comment]
The on-mob life change is :+1:
Not sure you need to "compensate" what I would consider a bugfix by making SalGlu better...I mean, it's pretty damn good already, seeing as you can have 1000u of this stuff in you, since you never OD on it.

**Comment by Zostroll** [comment]
This is a fundamental game mechanic.... It has been like this forever and without it I would die like 100% more. It's slow enough that you can kill someone with it in there system but good enough to let you fight a few xeno mobs. I see absolutely no reason for it to be removed. The only person that has any problems with it is you. One patch takes like 2 minutes tops to be metabolised even taking all of the brute kits its like 15 mins of constant healing and the only way to get more is through being a chemist or CMO. Removing this feature would be a mistake. Maybe make it metabolise slower healing quite a bit less say 350ish brute.

**Comment by ShadowDeath6** [comment]
having it heal someone from full brute crit is a retarded idea, and instant full heals is hugbox for miners considering they have legions souls as well, but at least adam is taking the high road and not buffing it to compensate for removing something

**Comment by BOBsHERE** [comment]
I see where your coming from with brute patches basically being synthflesh combined with bicardine. I prefer that they heal slightly over time though.

**Comment by ghost** [comment]
I give absolutely no fucks about any complaints, especially yours, Zostroll, as any disagreements absolutely indicates a fundamental misunderstanding with how the game is supposed to work.

This is a bug, plain and simple. Fuck off if you're complaining about 'i'd die without it.'

**Comment by Zostroll** [comment]
I made valid points if you want to ignore them it's on you but to anyone reading this with merge power's please don't wildly merge this. It's a major game change and if more than five people used github you would see that a large percentage of the server would be against this change.
